packages/experiments/src/components/UnifiedPicker/UnifiedPicker.tsx

```diff
@@ -242,9 +239,13 @@ export const UnifiedPicker = <T extends {}>(props: IUnifiedPickerProps<T>): JSX.
                   onClick={_onInputClick}
                   onInputValueChange={_onInputChange}
                   /* eslint-enable react/jsx-no-bind */
-                  aria-activedescendant={activeDescendant}
-                  aria-owns={isExpanded ? 'suggestion-list' : undefined}
-                  aria-expanded={isExpanded}
+                  aria-activedescendant={
+                    isSuggestionsShown && focusItemIndex >= 0
+                      ? 'FloatingSuggestionsItemId-' + focusItemIndex
+                      : undefined
+                  }
+                  aria-owns={isSuggestionsShown ? 'suggestion-list' : undefined}
```

Comment:
Somewhere in here I think we also need `aria-autocomplete='list'`

Reply:
Updated. Autofill was setting that value to both, but I believe you're correct and it should be list here